resources/ext-CodeSpecification.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-CodeSpecification" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-CodeSpecification" />
+  <name value="ExtCodeSpecification" />
+  <title value="ext CodeSpecification" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="Sometimes it is not possible to use the ValueSet from the functional description on the corresponding FHIR element. This happens oftentimes when the FHIR element has a _required_ binding with a FHIR core ValueSet that is incompatible with the ValueSet from the functional description. It can also happen when the FHIR element doesn't have a _CodeableConcept_ or _Coding_ datatype.&#xD;&#xA;&#xD;&#xA;In this case, the FHIR element may be augmented with this extension to communicate the code from the functional description. This information should always be _added_ to the normal use of the FHIR element to allow for maximum interoperability." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="rim" />
+    <uri value="http://hl7.org/v3" />
+    <name value="RIM Mapping" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="code" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="Coding" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="CodeableConcept" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="Address.country" />
+  </context>

```

Comment:
Dit is een waardeloze toevoeging, maar overgenomen uit de STU3 versie. Je zou kunnen beargumenteren dat er een custom extensie moet komen voor dit geval.

Reply:
Terwijl de beschrijving juist wel uitgaat van het scenario dat je de zib-code aan een string wil toevoegen. Liefst zou ik gewoon zien dat je deze extensie overal kan toepassen. Ja kan je ook voorstellen dat de zib ergens een waardelijst met "ja" en "nee" definieert die in FHIR op een valueBoolean terecht komt (of zo).
Zou je niet wegkomen met `type` = _fhirpath_ en `expression` = `true` of zo?

Reply:
De `type `= _fhirpath_ en `expression` =` $this` lijkt bij de Ext-Comment te werken... weet niet of dat de meest mooie oplossing is

Als we het hier nog iets generieker willen neerzetten kunnen we het string datatype toevoegen?

<context>
    <type value="element" />
    <expression value="string" />
  </context>

Reply:
Werkt dit zo? `expression` is een FhirPath-expressie toch?

Reply:
![image](https://user-images.githubusercontent.com/20518398/109190174-1caf1200-7795-11eb-805b-fbcf8e4505c5.png)

Yes, dit werkt zo.

Reply:
Maar als je "string" invult, wordt dat dan niet geïnterpreteerd als het (niet-bestaande) element "string" ipv een datatype?

Reply:
Volgens mij is dit besproken en nu ook toegevoegd :)